exercises/concept/weather-forecast/.docs/instructions.md

```diff
@@ -1,21 +1,15 @@
 # Instructions
 
-Being hired by a big weather forecast company, your job is to maintain their code base. Scrolling through the code, you find it hard to follow what the code is actually doing. Good thing you know just the right thing to do!
+Goblinocus is a country that takes its weather forecast very seriously! Since you are a renowned responsible proficient developer, they have asked you to write a program for them that can forecast the current weather condition of various cities in Goblinocus. You were busy at that time and asked one of your friends to do the job instead. After a while, the president of Goblinocus contacted you and said they do not understand a bit about your friend's code! When you check the code, you find out your friend did not act as a responsible programmer and there are no comments in the code!!! Therefore as a responsible person, you feel obligated to clarify the program so goblins can understand them as well!
 
 ## 1. Document package weather
 
-Write a package comment for `package weather` that describes its contents. The package comment should introduce the package and provide information relevant to the package as a whole.
+Since goblins are not as smart as you are, they forgot what the package should do for them! Please write a comment for `package weather` that describes its contents. The package comment should introduce the package and provide information relevant to the package as a whole.
 
-## 2. Document the CurrentCondition variable
+## 2. Document the CurrentCondition and CurrentLocation variables
 
-Write a comment for the package variable `CurrentCondition`.
+The president of Goblinocus is a bit paranoid and fears uncommented variables are used for destroying their country! Please clarify the usage of the package variables `CurrentCondition` and `CurrentLocation` and put the president's mind at ease. This should tell any user of the package what information the variables store, and what they can do with it.
 
-This should tell any user of the package what information the variable stores, and what they can do with it.
+## 3. Document the Forecast() function
```

Comment:
Changes to the instructions' titles must also be reflected in the [hints file](https://github.com/exercism/go/blob/main/exercises/concept/weather-forecast/.docs/hints.md). Can you please also apply this change there as well?